packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1720,8 +1720,29 @@ export class ApexDebug extends LoggingDebugSession {
 
       // log to console and notify client
       this.logEvent(message);
+      let reason = '';
+
+      // if breakpointid is found in exception breakpoint cache
+      // set the reason for stopped event to that reason
+      if (message.sobject.BreakpointId) {
+        const cache: Map<
+          string,
+          string
+        > = this.myBreakpointService.getExceptionBreakpointCache();
+        cache.forEach((value, key) => {
+          if (value === message.sobject.BreakpointId) {
+            // typerefs for exceptions will change based on whether they are custom,
+            // defined as an inner class, defined in a trigger, or in a namespaced org
+            reason = key
+              .replace('com/salesforce/api/exception/', '')
+              .replace('__sfdc_trigger/', '')
```

Comment:
Can we pull these two prefixes out to constants.ts so they can also be used in the tests?